status/flowcell.py

```diff
@@ -108,11 +108,36 @@ def get(self, flowcell_id):
             entry.value['plist'] = self._get_project_list(entry.value)
             # list of project_names -> to create links to project page and bioinfo tab
             project_names = {project_name: self._get_project_id_by_name(project_name) for project_name in entry.value['plist']}
+            # Prepare a summary table for total project yields in each lane
+            fc_project_yields = dict()
+            for lane_nr in sorted(entry.value.get('lanedata').keys()):
+                fc_project_yields_lane_list = []
+                lane_details = entry.value['lane'][lane_nr]
+                total_lane_yield = int(entry.value['lanedata'][lane_nr]['clustersnb'].replace(',',''))
+                unique_projects = list(set(i['Project'] for i in lane_details))
```

Comment:
I would avoid using `i` as a general purpose variable:
```suggestion
                unique_projects = list(set(item['Project'] for item in lane_details))
```